Sources/Public/Animation/AnimationView.swift

```diff
@@ -939,6 +955,87 @@ final public class AnimationView: AnimationViewBase {
     currentFrame = CGFloat(animation.startFrame)
   }
 
+  fileprivate func makeMainThreadAnimationLayer(for animation: Animation) -> MainThreadAnimationLayer {
+    MainThreadAnimationLayer(
+      animation: animation,
+      imageProvider: imageProvider,
+      textProvider: textProvider,
+      fontProvider: fontProvider)
+  }
+
+  fileprivate func makeCoreAnimationLayer(for animation: Animation) -> CoreAnimationLayer? {
+    do {
+      let coreAnimationLayer = try CoreAnimationLayer(
+        animation: animation,
+        imageProvider: imageProvider,
+        fontProvider: fontProvider,
+        compatibilityTrackerMode: .track)
+
+      coreAnimationLayer.didSetUpAnimation = { compatibilityIssues in
+        LottieLogger.shared.assert(
+          compatibilityIssues.isEmpty,
+          compatibilityIssues.compatibilityMessage)
+      }
+
+      return coreAnimationLayer
+    } catch {
+      LottieLogger.shared.assertionFailure("Encountered unexpected error \(error)")
```

Comment:
Could you include a comment detailing how we don't expect this case to occur since we're using the `track` mode, and this is programmer error if we hit this (I assume)?